symfony/plugins/orangehrmMaintenancePlugin/lib/service/AuthenticateVerifyService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: administrator
+ * Date: 6/9/18
+ * Time: 12:33 PM
+ */
+class AuthenticateVerifyService
+{
+
+    public function isCurrentPassword($userId, $password)
+    {
+        $systemUser = $this->getSystemUserDao()->getSystemUser($userId);
+        if (!($systemUser instanceof SystemUser)) {
+            return false;
+        }
+        $hash = $systemUser->getUserPassword();
+
+        if ($this->checkPasswordHash($password, $hash)) {
```

Comment:
return $this->checkPasswordHash($password, $hash)

Reply:
done